ajax/libs/elfinder/package.json

```diff
@@ -24,7 +24,7 @@
       {
         "basePath": "",
         "files": [
-          "js/**/+(elFinder|elfinder|quicklook.googledocs)*.+(js|map)",
+          "js/**/!(*encoding-japanese.min)*.+(html|js|map)",
```

Comment:
@nao-pon Thank you for your comment.
I finally know the `js/i18n/help/*.html`. But I think it not necessary for web developers because they can't use HTML tag.
And with the CDNJS link, `js/i18n/help/*.html` will show the raw code. I think end-user don't interested with the raw code of HTML.
Because CDNJS host a lot of files of libraries, if the file is necessary, we still want every file can be useful to people to use. But if the file is not necessary, we want to lessen the host disk loading.

Would you help us to remove `html|` part?
It will be great work.

Thank you very much.